docs/requirements.txt

```diff
@@ -1,57 +1,60 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.12
 # by the following command:
 #
-#    pip-compile --allow-unsafe --no-emit-index-url --output-file=docs/requirements.txt --strip-extras requirements-docs.in
+#    pip-compile --allow-unsafe --cert=None --client-cert=None --index-url=None --no-emit-index-url --output-file=docs/requirements.txt --pip-args=None --strip-extras requirements-docs.in
```

Comment:
(these extra arguments are because I use a "devpi-server" instance on localhost, they're just comments and harmless anyway)